javascript-modal/main.js

```diff
@@ -0,0 +1,18 @@
+// get class .modal and assign it to var modal
+// get element button and assign it to var button
+// create function to open modal when button is clicked
+// create NO button within modal to close modal when clicked
+
+var modal = document.querySelector('.modal');
+var button = document.querySelector('button');
+var close = document.querySelector('.close');
+
+button.addEventListener('click', function click() {
+  modal.style.display = 'block';
```

Comment:
This is using inline styling, which is far too strong and shouldn't be used. Instead you should prepare a class in your CSS that you can add or remove to hide or show the element.